hdt_monitor/settings/base.py

```diff
@@ -7,9 +7,19 @@
 https://docs.djangoproject.com/en/1.8/ref/settings/
 """
 
-import os
 import logging
-from .secure import SECURE_SETTINGS
+import os
+import time
+
+from dj_secure_settings.loader import load_secure_settings
+from icommons_common.logging import JSON_LOG_FORMAT, ContextFilter
+
+SECURE_SETTINGS = load_secure_settings()
+
+try:
+    from build_info import BUILD_INFO
+except ImportError:
+    BUILD_INFO = {}
```

Comment:
Would we want to follow this with something like the following for logging purposes somewhere?

```python
print(f"Build info: {BUILD_INFO}", flush=True)
```

Reply:
In some of the other projects we've added build info to the page footer so that one can easily tell what version of the container image you're looking at, but since this isn't really our tool I'm not inclined to change the page.